book/website/welcome.md

```diff
@@ -1,13 +1,20 @@
 (welcome)=
 # Welcome
 
-_The Turing Way_ is a handbook to reproducible, ethical and collaborative data science. 
+*Welcome to The Turing Way guides to reproducible, ethical and collaborative data science.*
```

Comment:
I think this phrasing might be confusing if we don't match the titles of the guides in the second half of the sentence.

E.g. Welcome to The Turing Way guides to Reproducible Research, Project Design, Communication, Collaboration, and Ethical Research.

Reply:
_The Turing Way_ is a handbook to reproducible, ethical and collaborative data science.
```suggestion
_The Turing Way_ is a community-led project to reproducible, ethical and collaborative data science.
```
I guess my reason to edit this line was to avoid the assumption that The Turing Way is only a handbook. Does this editing on your original work better?

Reply:
I'm curious to understand better what you mean by 'only a handbook'? 

'project to' doesn't work in my opinion. It would need to be 'project which aims to enable' or 'project where the goal is to teach' or something like that.

Reply:
The Turing Way is a book as well as a community.
I agree that the sentence needs more update, I edited it, but then again it is longer than what you suggested.

Reply:
what could be the best filler for `<>`?

Reply:
> The Turing Way is a book as well as a community-led project that does more than just writing chapters in a handbook.

Thanks for the clarification. I agree.
However, I think it would be reasonable in this first line to welcome people to the guide/handbook and then to flow on with more detail about the project in a broader sense in the following sentences.

I would suggest:
```suggestion
*Welcome to The Turing Way handbook to reproducible, ethical and collaborative data science.*
```
so long as it is quickly followed by 'the TR project is open XYZ and community driven, and aims to foster/engage/create a community etc etc'

Reply:
I like the suggestion on the following line clarifying that. I am committing the suggested change.